packages/reports/app/styles/navi-reports/components/filter-collection.less

```diff
@@ -41,12 +41,13 @@
     &__operator,
     &__field {
       flex: 1;
-      margin: @filter-collection-column-margin;
+      margin: @filter-collection-column-margin 0;
```

Comment:
do these changes still look okay in the reports filter-collection?

Reply:
yep, i've double triple checked.